src/compressor.js

```diff
@@ -65,7 +73,7 @@ define(function (require) {
 
 
   /**
-   * set attack w/ time ramp or get current attack
+   * Get current attack or set value w/ time ramp
    * @method attack
```

Comment:
Definitely include the descriptions you have above for `process` for all of these methods, they're really helpful.

Also, all of these methods should include info about the method's `@param`'s. Optional param names can be in square brackets like `[attack]`

`@param {Number} [attack] Time in seconds, between 0 and 1. Default 0.03.`